server/protocols/jmap/src/test/java/org/apache/james/jmap/model/mailbox/MailboxTest.java

```diff
@@ -96,4 +96,65 @@ public void parentIdDefaultValueIsEmpty() {
 
         assertThat(mailbox.getParentId()).isEmpty();
     }
+
+    @Test
+    public void hasRoleShouldReturnFalseWhenMailboxEmptyRole() {
+        Mailbox mailbox = Mailbox.builder()
+            .id(InMemoryId.of(0))
+            .name("name")
+            .build();
+
+        assertThat(mailbox.hasRole(Role.OUTBOX)).isFalse();
+    }
+
+    @Test
+    public void hasRoleShouldReturnFalseWhenMailboxDoesNotHaveRole() {
+        Mailbox mailbox = Mailbox.builder()
+            .id(InMemoryId.of(0))
+            .name("name")
+            .role(Optional.of(Role.DRAFTS))
+            .build();
+
+        assertThat(mailbox.hasRole(Role.OUTBOX)).isFalse();
+    }
+
+    @Test
+    public void hasRoleShouldReturnTrueWhenMailboxHasRole() {
+        Mailbox mailbox = Mailbox.builder()
+            .id(InMemoryId.of(0))
+            .name("name")
+            .role(Optional.of(Role.DRAFTS))
+            .build();
+
+        assertThat(mailbox.hasRole(Role.DRAFTS)).isTrue();
+    }
+
+    @Test
+    public void hasSystemRoleShouldReturnFalseWhenMailboxHasNotRole() throws Exception {
+        Mailbox mailbox = Mailbox.builder()
+            .name("mailbox")
+            .id(InMemoryId.of(0))
+            .build();
```

Comment:
If you want to be consistant with you previous test you should add a empty line before all your assertThat